functions/uploader/entities/entities.js

```diff
@@ -49,7 +49,7 @@ function postEntitiesToDF (entityid, entities, first) {
   }
   return getAccessToken()
     .then(accesstoken => {
-      debug(util.inspect(JSON.stringify({'entities': data}), false, null));
+      debug(util.inspect({'entities': data}, false, null));
```

Comment:
what is the sense to log data only after `getAccessToken()`? And you can use `'entities'` without quotes here, the same for any object literal notation which you use